protobuf/coreapi/api.proto

```diff
@@ -243,10 +243,17 @@ message DeployServiceReply {
     Type type = 2;      // Type of status message.
   }
 
+  message Service {
+    string SID = 1;   // `SID` is the service ID.
+    string hash = 2;  // `hash` is the service hash.
+  }
+  
+  reserved 1, 3;
```

Comment:
We should use this `reserved` everytime we remove a parameter from a proto definition.
It prevents future use of the number.

Reply:
I think we could use reserved when we reach alpha or beta stage, for now, let's make use of where we are right now :) But in the future we should use `reserved` of course

Reply:
Agree with @krhubert here, let's not add complexity for now especially that this will be the only api with that. We should do it everywhere or nowhere to have consistency. If we want to do this can be done in another PR for all the apis. I would remove this part for this PR.

Reply:
```suggestion
```

Reply:
Removed.